### PR TITLE
Xamarin AEPAssurance 1.0.0

### PR DESCRIPTION
**Pull request by PravinPK**
Whats included in this changes
- IOS Assurance framework updated to 1.1.3
- Android Assurance framework updated to 1.0.3
- Xamarin Android library depdends on contain layout package
- Android target versions for SDK, test app and unit test are set to Android 10
- Unit test updates

### CODE REVIEW

File: src/AEPAssuranceiOSUnitTests/mono_crash.11a67480d0.0.json [hunk 1]
```diff
@@ -0,0 +1,731 @@
+{
```
Comment (cacheung) [inline src/AEPAssuranceiOSUnitTests/mono_crash.11a67480d0.0.json:1]: This is a test file?

File: Makefile [hunk 2]
```diff
@@ -24,3 +24,22 @@ release:
 	mkdir bin
 	cp src/Adobe.AEPAssurance.Android/bin/Debug/*.nupkg ./bin
 	cp src/Adobe.AEPAssurance.iOS/bin/Debug/*.nupkg ./bin
+
+
+AEPASSURANCE_SDK_PATH = ./acp-sdk
```
Comment (cacheung) [inline Makefile:29]: Other extension add acp-sdk/ to .gitignore just to be safe.
https://github.com/adobe/xamarin-acpanalytics/blob/master/.gitignore#L345

**Review by cacheung — COMMENTED** [review]
We don't need the "binaries" folder in the root anymore.

File: Makefile (hunk 2 above)
Comment (PravinPK) [reply Makefile:29]: will do

File: src/AEPAssuranceiOSUnitTests/mono_crash.11a67480d0.0.json (hunk 1 above)
Comment (PravinPK) [reply src/AEPAssuranceiOSUnitTests/mono_crash.11a67480d0.0.json:1]: These are files that contain mono crash dump information... It is safe for them to be deleted an not checked In
Let me do that.. thanks for identifying them

**Review by cacheung — APPROVED** [review]

**Pull request merged**